front/src/applications/operationalStudies/views/Scenario/components/ImportTimetableItem/ImportTimetableItem.tsx

```diff
@@ -11,11 +11,11 @@ import ImportTimetableItemConfig from './ImportTimetableItemConfig';
 import ImportTimetableItemTrainsList from './ImportTimetableItemTrainsList';
 
 type ImportTimetableItemProps = {
-  scenario: ScenarioResponse;
   upsertTimetableItems: (timetableItems: TimetableItem[]) => void;
+  scenario: ScenarioResponse;
```

Comment:
You can remove the scenario from the props since you will get it from the context